source/release-notes/1.6.txt

```diff
@@ -0,0 +1,145 @@
+.. _c2c-release-notes-1.6:
+
+===================================
+Release Notes for mongosync 1.6
+===================================
+
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _1.6.0-c2c-release-notes:
+
+1.6.0 Release
+-------------
+
+**September 22, 2023**
+
+Exclusion Filters
+~~~~~~~~~~~~~~~~~
+
+Starting in 1.6.0, the :ref:`c2c-api-start` API endpoint now supports the
+use of exclusion filters through the ``excludeNamespaces`` parameter.
+
+For more information, see :ref:`c2c-filtered-sync`.
+
+Regular Expression Filters
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Both inclusion and exclusion filters in :ref:`c2c-filtered-sync` now 
+support matching databases and collections using Regular Expressions 
+with the ``databaseRegex`` and ``collectionRegex`` fields.
+
+For more information, see :ref:`c2c-filter-regex`.
+
+Exclusion Filters
+~~~~~~~~~~~~~~~~~
+
+Starting in 1.6.0, the :ref:`c2c-api-start` API endpoint now supports the
+use of exclusion filters through the ``excludeNamespaces`` parameter.
+
+For more information, see :ref:`c2c-filtered-sync`.
+
+Regular Expression Filters
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Both inclusion and exclusion filters in :ref:`c2c-filtered-sync` now 
+support matching databases and collections using Regular Expressions 
+with the ``databaseRegex`` and ``collectionRegex`` fields.
+
+For more information, see :ref:`c2c-filter-regex`.
```

Comment:
Both of these sections appear twice in the notes; one of them can be deleted.

Reply:
Good catch.  Looks like Git confusion from the cherry-pick.

Reply:
The Exclusion Filters section is still there twice.